Week2/Section1_w2/Exercise1/Source1.cpp

```diff
@@ -10,18 +10,18 @@
 class FFirstClass
 {
 public:
-	int value;
+	int Value;
 
-	FFirstClass(int value)
+	FFirstClass(int NewValue)
 	{
-		//I create this variable to identify the diferent instances. This was interesting I try using "this.value" but VS correct me and use "this->value" :)
-		this->value = value;
-		std::cout << "I'm the Constructor " << this->value << std::endl;
+		//I create this variable to identify the diferent instances. This was interesting I try using "this.Value" but VS correct me and use "this->Value" :)
```

Comment:
No need for this comment since you are not using `this` anymore. The -> operator will be explained next week :)

Reply:
Deleted 👌